### PR TITLE
Sample ts

### PR DESCRIPTION
**Pull request by unkrich**
## Pull request type

Please check the type of change your PR introduces:

- [x] Feature

## Issue

Z scores had a wrong index occuring this is fixed.

This moves away from backfilling based on a timestamp and instead sampling at the time of the job running, and then the job runs every hour

### CODE REVIEW

File: src/ingestion/transformers/monosi/zscores.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def _transform(cls, metrics):
             for metric in groups[table]:
                 for column_name in groups[table][metric]:
                     group_metrics = groups[table][metric][column_name]
+                    print(group_metrics)
```
Comment (iporollo) [inline src/ingestion/transformers/monosi/zscores.py:91]: remove

File: src/ui/src/pages/app/monitors/Index/components/MonitorsTable/index.tsx [hunk 2]
```diff
@@ -38,11 +38,12 @@ const MonitorsTable: React.FC<{
...
-        if (row.timestamp_field === null) {
-                return (
-                  <span className="badge rounded-pill bg-danger">Disabled</span>
-                )
-        } else if (row.metrics == 0) {
+        // if (row.timestamp_field === null) {
+        //         return (
+        //           <span className="badge rounded-pill bg-danger">Disabled</span>
+        //         )
+        // } else
```
Comment (iporollo) [inline src/ui/src/pages/app/monitors/Index/components/MonitorsTable/index.tsx:45]: Let's just remove this if it will never be `null` now?

**Review by iporollo — COMMENTED** [review]
Should we keep both timestamp window & sampling functionality rather than replacing?

File: src/ui/src/pages/app/monitors/Index/components/MonitorsTable/index.tsx (hunk 2 above)
Comment (unkrich) [reply src/ui/src/pages/app/monitors/Index/components/MonitorsTable/index.tsx:45]: the timestamp field logic is still going to apply in the near future I would imagine

**Review by iporollo — APPROVED** [review]

**Pull request merged**